scripts/rollup/rollup.config.js

```diff
@@ -43,6 +43,10 @@ if (packageName === '@lwc/synthetic-shadow') {
 }
 
 const onwarn = ({ code, message }) => {
+    if (code === 'INVALID_ANNOTATION') {
+        // Not sure why the PURE annotations are broken.
+        return;
+    }
```

Comment:
A TODO linking to a GitHub issue would be helpful here (e.g. to explain what annotations are and why they're invalid 🙂 ).

Reply:
This isn't broken anymore (and I didn't dig into why it was originally broken). So I'll revert this bit.